node-manager/src/nodemanager.rs

```diff
@@ -17,10 +22,23 @@ use crate::{
 pub struct NodeManager {
     mnemonic: Mnemonic,
     wallet: MutinyWallet,
+    node_storage: Mutex<NodeStorage>,
     ws_write: Arc<Mutex<SplitSink<WebSocket, Message>>>,
     counter: usize,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct NodeStorage {
+    pub nodes: Vec<NodeIndex>,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct NodeIndex {
+    pub id: String,
+    pub pubkey: String,
```

Comment:
can we make this `PublicKey` instead of `String`? Not sure if it has `Serialize, Deserialize`

Reply:
Looks like it does serialize to string correctly, I'll do that

Reply:
done